includes/admin/settings/class-settings-display.php

```diff
@@ -87,6 +87,17 @@ public function get_settings() {
 								'optional' => __( 'Optional', 'give' ),
 							],
 						],
+						[
+							'name'    => __( 'Last Name Required', 'give' ),
+							'desc'    => __( 'Do you want to force the Last Name field to be required?', 'give' ),
+							'id'      => 'last_name_field_required',
+							'type'    => 'radio_inline',
+							'default' => 'optional',
+							'options' => [
+								'required' => __( 'Required', 'give' ),
+								'optional' => __( 'Optional', 'give' ),
+							],
+						],
 						[
 							'name'    => __( 'Anonymous Donations', 'give' ),
 							'desc'    => __( 'Do you want to provide donors the ability mark themselves anonymous while giving. This will prevent their information from appearing publicly on your website but you will still receive their information for your records in the admin panel.', 'give' ),
```

Comment:
Noticed a missing question mark on line 103. "Do you want to provide donors the ability to mark themselves anonymous while giving?"